Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -346,6 +351,26 @@ - (BOOL)handleDeepLink:(NSURL *)url {
     return handled;
 }
 
+- (BOOL)continueUserActivity:(NSUserActivity *)userActivity {
+    if ([userActivity.activityType isEqualToString:NSUserActivityTypeBrowsingWeb]) {
+        self.preferenceHelper.universalLinkUrl = [userActivity.webpageURL absoluteString];
```

Comment:
so this looks like we will try to handle any universal link!! even non-Branch ones. With spotlight, at least we look for the branch prefix.